opentracing-shim/src/main/java/openconsensus/opentracingshim/SpanBuilderShim.java

```diff
@@ -59,19 +61,59 @@ public SpanBuilder ignoreActiveSpan() {
 
   @Override
   public SpanBuilder withTag(String key, String value) {
-    // TODO
+    if (key == "span.kind") {
```

Comment:
Btw, string comparison does not work through `==` in Java - you have to use `equals()` (we don't have anything C#'s operator overload ;) )

Reply:
thanks, fixed